cocotb/binary.py

```diff
@@ -144,12 +147,16 @@ def __init__(self, value=None, n_bits=None, bigEndian=True,
             self.assign(value)
 
     def assign(self, value):
-        """Decides how best to assign the value to the vector
+        """Decides how best to assign the value to the vector.
 
         We possibly try to be a bit too clever here by first of
         all trying to assign the raw string as a binstring, however
         if the string contains any characters that aren't 0, 1, X or Z
-        then we interpret the string as a binary buffer...
+        then we interpret the string as a binary buffer.
+
+        Args:
+            value (str or int or long):
```

Comment:
I think `long` no longer (pun intended) exists in python, or at least not in standard python.  I think it exists in some python modules, but not sure this is what is meant here.

Reply:
Not true, it still exists in Python 2